common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -262,9 +265,17 @@ protected final AuthorizationRequest.Builder initializeAuthorizationRequestBuild
             Logger.error(TAG, "correlation id from diagnostic context is not a UUID", ex);
         }
 
-        builder.setClientId(parameters.getClientId())
-                .setRedirectUri(parameters.getRedirectUri());
 
+        if (parameters.hasNestedAppParameters()) {
+            // is NAA scenario, set all NAA params for auth flow
+            builder.setBrkRedirectUri(parameters.getRedirectUri())
```

Comment:
See my comments in other places about naming

Reply:
My overall recommendation, let's pick one "new" parameter name. So if you want to use new name "child" for the child app then let's use that in all places. Or if you want to rather use new word "brkr" for the hub then let's do that in all places. Rather than introducing multiple new names.

My preference is the introduction of "child" to denote child apps. I suggest we use that everywhere. 

I know you may have to use different naming for what you send to eSTS. My suggestion is that you achieve that via SerializedNames

Reply:
responded to this in prev similar comment above